### ISSUE TITLE
Error while installing in ember app

### ISSUE THREAD

**Issue by Kilowhisky** [lifegadget/ui-slider#9]
I installed the plugin using `ember install ui-ember-slider` and everything went through fine. But when i went to run the app with ember serve i get this error. 

```
Serving on http://localhost:4200/app/
ENOENT: no such file or directory, stat 'C:\Projects\GIT\app\tmp\concat_with_maps-input_base_path-iM5eFq7U.tmp\0\bower_components\seiyria-bootstrap-slider\js\bootstrap-slider.js'
Error: ENOENT: no such file or directory, stat 'C:\Projects\GIT\app\tmp\concat_with_maps-input_base_path-iM5eFq7U.tmp\0\bower_components\seiyria-bootstrap-slider\js\bootstrap-slider.js'
```

Maybe related to? https://github.com/broccolijs/broccoli-funnel/issues/53

**Comment by jweakley** [comment]
We are getting that error too. I don't see a blueprint for the app that installs `seiyria-bootstrap-slider`, but I see index.js trying to pull it in.

**Comment by jweakley** [comment]
The current work around is to run `bower install seiyria-bootstrap-slider --save` so that it is available for it.

**Comment by ksnyde** [comment]
Oops, sorry guys. I'll aim to get this fixed today.

**Comment by ksnyde** [comment]
This should be fixed now in npm version `0.2.2`. Thanks for pointing this out. I'll close this but please re-open if it's still a problem for either of you.

**Comment by jweakley** [comment]
:+1: that fix worked for me. Thanks for the quick update!
